src/zentables/zentables.py

```diff
@@ -735,6 +735,8 @@ def mean_sd_table(
         count = pivot.xs("count", axis=1).astype("Int64").fillna(0)
         mean = pivot.xs("mean", axis=1)
         std = pivot.xs("std", axis=1)
+        # issues can arise when the shape is inconsistent because some groupings are entirely empty
+        assert count.shape == mean.shape == std.shape, "Ensure that all categories have at least some entry"
```

Comment:
You should raise a `ValueError` or somesuch. In general, `AssertionError`s are not meant to be caught, but you may wish to catch this one.

Reply:
Done.